.github/workflows/packages.yml

```diff
@@ -114,10 +114,8 @@ jobs:
             -
                 name: Run PHPUnit
                 working-directory: "src/Sylius/${{ matrix.package }}"
-                run: |
-                    if [[ -e "phpunit.xml.dist" && -e "phpunit.xml" ]]; then
-                        vendor/bin/phpunit --colors=always
-                    fi
+                if: hashFiles('phpunit.xml.dist') != '' || hashFiles('phpunit.xml') != '' 
```

Comment:
Nice!

Reply:
lets see if it works. I have another solution under my sleeve if not: https://github.community/t/run-step-if-file-exists/16445/6